tests/internal/test_forksafe.py

```diff
@@ -288,61 +288,89 @@ def fn():
     assert exit_code == 42
 
 
-# FIXME: subprocess marks do not respect pytest.mark.skips
-if sys.version_info < (3, 11, 0):
-
-    @pytest.mark.subprocess(
-        out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"),
-        err=None,
-        env=dict(_DD_TRACE_GEVENT_HUB_PATCHED="true"),
-    )
-    def test_gevent_reinit_patch():
-        import os
-        import sys
+@pytest.mark.subprocess(
+    out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"), err=None
+)
+def test_gevent_gunicorn_behaviour():
+    # ---- Emulate sitecustomize.py ----
```

Comment:
I think it might be worth adding a comment to provide a little more context.
```suggestion
    # emulate how sitecustomize.py cleans up imported modules
    # to avoid problems with threads/forks that we saw previously
    # when running gunicorn with gevent workers
```

Reply:
On second thought, do we need this test? Considering we have the gunicorn test scenarios that accomplish the same thing (unless I'm mistaken).

Reply:
I think this was added to have a quick way of telling if we were breaking the support we had for `gevent`. @emmettbutler I think @Yun-Kim is right that this is now covered by your new tests. WDYT?

Reply:
Sure, I don't mind if we remove it @P403n1x87